### DISCUSSION TITLE
Port Feature Requests from the Wiki

### DISCUSSION THREAD

**Discussion by BenjamenMeyer** [vegastrike/Vega-Strike-Engine-Source#983] [General]
Feature requests were tracked in the wiki, there's quite a few and a lot of details. It'd be helpful to actually pull them over here and mark them as such.
https://wiki.vega-strike.org/FAQ:Feature_Requests

**Comment by royfalk** [comment]
That’s a good idea. Thank you.

**Comment by BenjamenMeyer** [comment]
This probably should be a discussion instead.

**Comment by BenjamenMeyer** [comment]
Review the conversation at https://forums.vega-strike.org/viewtopic.php?f=27&t=11136&p=170049#p170049 on previous plans for 0.6.0 and a guide from 0.5.0 to 0.6.0 to see what is still applicable and file appropriate issues.
